src/index.ts

```diff
@@ -3,20 +3,28 @@ import {install as installImageLink, uninstall as uninstallImageLink} from './pa
 import {install as installLink, uninstall as uninstallLink} from './paste-markdown-link'
 import {install as installTable, uninstall as uninstallTable} from './paste-markdown-table'
 import {install as installText, uninstall as uninstallText} from './paste-markdown-text'
+import {
+  installBefore as installUnformatted,
+  installAfter as installUnformattedAfter,
+  uninstall as uninstallUnformatted
+} from './paste-keyboard-shortcut-helper'
 
 interface Subscription {
   unsubscribe: () => void
 }
 
 function subscribe(el: HTMLElement): Subscription {
+  installUnformatted(el)
   installTable(el)
   installImageLink(el)
   installLink(el)
   installText(el)
   installHTML(el)
+  installUnformattedAfter(el)
```

Comment:
I can't think of something better, but I don't like the `installUnformatted` and `installUnformattedAfter` names 😅 
I don't think they explain what they are actually doing

Reply:
addressed in https://github.com/github/paste-markdown/pull/59/commits/fc32d4d489c236cb705b019537d8f878a67d4196